### DISCUSSION TITLE
Feature Request: filter files in "Files changed" view to files with an (active) comment

### DISCUSSION THREAD

**Discussion by Feyko** [community/community#28259] [Pull Requests]
All in the title! Would be nice to be able to only see files that have active comment. Specifically useful for applying suggestions in batch

**Comment by budak7273** [comment]
This would be quite nice. The 'Conversations' tab does not fulfil this need because it doesn't allow you to batch apply suggestions.

The 'workaround' for now seems to be either marking files as viewed to collapse them (which can accidentally hide open discussions) or using the Conversations jump dropdown in the Files tab.
